### DISCUSSION TITLE
Should this be used in place of hayes0724 / shopify-webpack-themekit?

### DISCUSSION THREAD

**Discussion by adrianocr** [hayes0724/shopify-packer#51] [Q&A] [answered]
There seems to be a lot of overlap between the two and I'm not fully understanding where one should take the place of the other?

**Comment by adrianocr** [comment]
Seems like hayes0724/shopify-webpack-themekit is more or less just a pre-defined theme type of setup whereas hayes0724/shopify-packer is trying to be a Slate replacement.

**Comment by hayes0724** [comment, marked as answer]
`shopify-webpack-themekit` is a webpack setup I used for many years (pre Slate) where everything (build tools, theme, etc) are all in one repo. When Slate was released there were several features I liked and wanted to add to `shopify-webpack-themekit`. Installing as a global node module that can be configured in a separate repo was a key feature that required a complete rebuild of the project. I created `shopify-packer` for this purpose and maintain both. `shopify-webpack-themekit` receives backports from `hayes0724/shopify-packer` since they are mostly compatible.

**hayes0724/shopify-packer**
- Drop in replacement for Slate (good for team projects where other members have Slate experience)
- Build tools are separate from theme (`@hayes0724/packer-blank-theme`)
- Easier to update existing projects
- Easy to use with existing themes
- Base theme is separate and not required
- CLI tools for common tasks (theme create, list, init, download, zip)

**hayes0724/shopify-webpack-themekit**
- Single repo for build and project
- Receives backports from packer
- Easier to edit build directly
- Harder to update previous build portion projects (since build and project are in the same repo)
- Hard to work with existing sites
- No CLI Tools
